src/components/Summary/AlterationSummary.vue

```diff
@@ -168,36 +168,36 @@ export default class AlterationSummary extends Mixins(CommonMixin, DateMixin, Fi
   /** Prop to perform validation. */
   @Prop() readonly pleaseValidate: boolean
 
-  private get isFutureEffective (): boolean {
```

Comment:
My linter says these aren't used when they are private, so I removed it. Here and elsewhere.